docs/source/tutorials/odsc_tutorial/remote_local.rst

```diff
@@ -0,0 +1,74 @@
+Launching Experiments Remotely
+==============================
+
+As a machine learning practitioner, you operate in a highly competitive
+landscape. Your success depends to a large extent on whether you can *decrease
+the time to the next decision*. In this section, we discuss one important
+approach, namely how to increase the *number of experiments run in parallel*.
+
+Launching our Study
+-------------------
+
+Here is how we specified and ran experiments of the
+`study above <comparison.html#a-comparative-study>`__. First, we specify a
+script for launching experiments locally:
+
+.. literalinclude:: ../../../../benchmarking/nursery/odsc_tutorial/transformer_wikitext2/local/hpo_main.py
+   :caption: transformer_wikitext2/local/hpo_main.py
+   :start-after: # permissions and limitations under the License.
+
+This is very simple, as most work is done by the generic
+:func:`benchmarking.commons.hpo_main_local.main`. Note that ``hpo_main_local``
+needs to be chosen, since we use the local backend.
+
+This local launcher script can be used to configure your experiment, given
+additional command line arguments, as is explained in detail
+`here <../benchmarking/bm_simulator.html#specifying-extra-arguments>`__.
+
+You can use ``hpo_main.py`` to launch experiments locally, but they'll run
+sequentially, one after the other, and you need to have all dependencies
+installed locally. A second script is needed in order to launch many
+experiments in parallel:
+
+.. literalinclude:: ../../../../benchmarking/nursery/odsc_tutorial/transformer_wikitext2/local/launch_remote.py
+   :caption: transformer_wikitext2/local/launch_remote.py
+   :start-after: # permissions and limitations under the License.
+
+Once more, all the hard work in done in
+:func:`benchmarking.commons.launch_remote_local.launch_remote`, where
+``launch_remote_local`` needs to be chosen for the local backend. Most important
+is that our previous ``hpo_main.py`` is specified as ``entry_point`` here. Here is
+the command to run all experiments of our study in parallel:
+
+.. code-block:: bash
+
+   python benchmarking/nursery/odsc_tutorial/transformer_wikitext2/local/launch_remote.py \
+     --experiment_tag odsc-1 --benchmark transformer_wikitext2 --num_seeds 10
+
+* This command launches 40 SageMaker training jobs, running 10 random repetitions
+  (seeds) for each of the 4 methods specified in ``baselines.py``.
+* Each SageMaker training job uses one ``ml.g4dn.12xlarge`` AWS instance. You can
+  only run all 40 jobs in parallel if your resource limit for this instance type
+  is 40 or larger. Each training job will run a little longer than 5 hours, as
```

Comment:
Nit: consider optionally adding a follow-up comment to the effect of: "it's easy to request a limit increase request for your AWS account if you want to run more training jobs in parallel."